torch/csrc/inductor/aoti_package/model_package_loader.cpp

```diff
@@ -64,8 +64,8 @@ std::string create_temp_dir() {
 
 namespace torch::inductor {
 
-const nlohmann::json& AOTIModelPackageLoader::load_json_file(
-    std::string json_path) {
+namespace {
+const nlohmann::json& load_json_file(std::string json_path) {
```

Comment:
Unrelated to this change, but in general one should not return reference to an object that was allocated on stack, as it'll likely gets corrupted (and I though we have a clang-tidy warning against that)